scripts/sysman/powerDomainProperties.yml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Power Domain Properties"
+version: "1.8"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Power Domain Properties Name"
+version: "1.8"
+name: $S_POWER_DOMAIN_PROPERTIES_EXP_NAME
+value: '"$XS_extension_power_domain_properties"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Power Domain Properties Extension Version(s)"
+version: "1.8"
+name: $s_power_domain_properties_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension structure for providing power domain information associated with a power handle"
```

Comment:
Could we mention the API that is used to retrieve this ?

Reply:
Why we need separate structure? Couldnt we simply add a new field like type/domain in existing zes_power_properties_t?

Reply:
thanks @jitendrasharma1989 added detail on how to retrieve the struct.

>Why we need separate structure? Couldnt we simply add a new field like type/domain in existing zes_power_properties_t?

No as that would break the ABI